docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -306,7 +306,7 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
 You can see that the Container was terminated because of `reason:OOM Killed`,
 where `OOM` stands for Out Of Memory.
 
-## Local ephemeral storage (alpha feature)
+## Local ephemeral storage (beta feature for version 1.10 and above)
 
```

Comment:
Can we remove the parentheses completely and

- add `{% include feature-state-beta.md %}` in a separate line
- propose this to release-1.10 branch